app/assets/src/components/visualizations/table/base_table.scss

```diff
@@ -134,3 +134,31 @@
     display: block;
   }
 }
+
+.selectableColumnClassName {
+  // LUCIA: We pass in the "selectableCell" className to
+  // our checkbox to modify the enclosing div as follows;
+  // this creates a large, clickable "button" the height of
+  // the row, making samples easier to select.
+  .selectableCell {
+    position: absolute;
+    overflow: visible;
+    height: 100%;
+    width: 50px;
+    margin: -11px -10px -10px -10px;
```

Comment:
Why margin vs. padding?

Reply:
Margin here increases the size of the `div` here to cover the entire clickable area. No such thing as negative padding.